include/RED4ext/Process-inl.hpp

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/Process.hpp>
+#endif
+#include <RED4ext/Relocation.hpp>
+
+// This file has been generated by zoltan (https://github.com/jac3km4/zoltan)
+
+#define Process_CloseHandles_Addr 0x2C23B30
+#define Process_Execute_Addr 0x2C23D70
+#define Process_Execute_0_Addr 0x2C23DD0
+#define Process_GetExitCode_Addr 0x2C23B60
+#define Process_Process_Addr 0x2C23AA0
+#define Process_ReadFromPipe_Addr 0x2C23B90
+#define Process_Terminate_Addr 0x2C24040
+#define Process_WaitUntilCompleted_Addr 0x2C240B0
+
+inline bool RED4ext::Process::CloseHandles()
```

Comment:
```suggestion
RED4EXT_INLINE  bool RED4ext::Process::CloseHandles()
```
Same for the others.